pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -0,0 +1,26 @@
+---
+title: Star Wars
+theme: responsive
+---
+
+%link{:href=>"/css/starwars.css", :rel=>"stylesheet"}
+
+.container_responsive.mobile-pad
```

Comment:
@caleybrock since this has a lot of the same responsive needs as the big carousel at code.org/learn, i'm considering doing that first and then reusing it here.